contracts/hardhat.config.ts

```diff
@@ -105,7 +105,7 @@ let config = {
     disambiguatePaths: false,
   },
   mocha: {
-    timeout: 100000,
+    timeout: 150000,
```

Comment:
![this is fine](https://github.com/smartcontractkit/chainlink/assets/5789419/e486fa17-ff4a-4d4f-bf6c-22e7890483cd)